.drone.yml

```diff
@@ -104,13 +106,41 @@ steps:
   - name: deploy to test
     <<: *deploy
     settings:
+       values_file: ./helm/values/test-values.yaml
        namespace: callisto-test
        kube_token:
          from_secret: test_kube_token
+    when:
+      event: 
+        - push
+      branch: 
+        - main
+    depends_on:
+      - deploy to dev
+
+  - name: check dev up
+    pull: if-not-exists        
+    image: &alpine_curl_image alpine/curl:3.14
+    commands: 
+    - chmod +x ./scripts/is-url-up.sh
+    - ./scripts/is-url-up.sh https://web.dev.callisto-notprod.homeoffice.gov.uk/health
+    when:
+      event: 
+        - push
+      branch: 
+        - main
+    depends_on:      
+      - deploy to test
```

Comment:
It does depend on deploy to dev, as deploy to test depends on deploy to dev

Reply:
I was trying to avoid a wait for dev to come up and pass the health check, in the meantime test could have been deploying because the dev deployment was a success...happy to change if you think it's better to wait

Reply:
Could they run simultaneously if they both depend on deploy to dev?